src/5e-SRD-StartingEquipment.json

```diff
@@ -6,60 +6,73 @@
-			"item": { "url": "/api/equipment/flail", "name": "Flail"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/glaive", "name": "Glaive"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/greataxe", "name": "Greataxe"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/greatsword", "name": "Greatsword"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/halberd", "name": "Halberd"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/lance", "name": "Lance"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/longsword", "name": "Longsword"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/maul", "name": "Maul"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/morningstar", "name": "Morningstar"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/pike", "name": "Pike"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/rapier", "name": "Rapier"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/scimitar", "name": "Scimitar"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/shortsword", "name": "Shortsword"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/trident", "name": "Trident"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/war-pick", "name": "War pick"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/warhammer", "name": "Warhammer"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/whip", "name": "Whip"}, "quantity": 1}]
-	}],
-	"choice_2": [{
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/handaxe", "name": "Handaxe"}, "quantity": 2}]
-	}, {
-		"choose": 1,
-		"type": "equipment",
-		"from": [{
-			"item": { "url": "/api/equipment/club", "name": "Club"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/dagger", "name": "Dagger"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/greatclub", "name": "Greatclub"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/handaxe", "name": "Handaxe"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/javelin", "name": "Javelin"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/light-hammer", "name": "Light hammer"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/mace", "name": "Mace"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/quarterstaff", "name": "Quarterstaff"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/sickle", "name": "Sickle"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/spear", "name": "Spear"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/crossbow-light", "name": "Crossbow, light"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/dart", "name": "Dart"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/shortbow", "name": "Shortbow"}, "quantity": 1}, {
-			"item": { "url": "/api/equipment/sling", "name": "Sling"}, "quantity": 1}]
-	}],
+		"item": { "url": "/api/equipment/javelin", "name": "Javelin"}, "quantity": 4}
+	],
+	"choices": [
```

Comment:
Should be named "starting_equipment_options"

Reply:
Done!